apps/cookbook/src/app/examples/card-example/examples/outline.ts

```diff
@@ -0,0 +1,21 @@
+import { Component } from '@angular/core';
+
+const config = {
+  selector: 'cookbook-card-example-outline',
+  template: `<kirby-card [hasPadding]="true" variant="outlined">
+    <strong>This is an outlined card</strong>
```

Comment:
Consider keeping naming in sync with UX:
```suggestion
    <strong>This is an outline card</strong>
```